frontend/src/components/Graph.tsx

```diff
@@ -21,13 +23,18 @@ let timee = 0
 let posx = 0
 let posy = 0
 let log = 1
+interface TimeBarData {
+  date: string
+  value: string
+}
```

Comment:
Move this to src/interfaces/ApiData.d.ts

Reply:
Fixed :)